### DISCUSSION TITLE
Why is my github education temporary?

### DISCUSSION THREAD

**Discussion by wheeleryanli** [community/community#138665] [GitHub Education]
### Hi everyone,

I received my github education benefits on September 11th, but today I suddenly realized I need to apply for github education again! I reapplied again, why is that? What do I need to do to fix this?
![20240915103130](https://github.com/user-attachments/assets/33150e20-9832-4f83-afc3-ae41008c3562)
![20240915103203](https://github.com/user-attachments/assets/99930bef-585d-4511-84da-89011250c3a4)

**Comment by davevad93** [comment]
Hi @wheeleryanli ,

That's odd because normally once approved, the status label turns green showing the message as shown in your screenshot and then when benefits are available the label turns purple and says "`Congrats! Your request for @wheeleryanli was approved and your benefits are now available.`". Also, the benefits last for two years upon approval.

If it happens again, I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Comment by LabyKreative** [comment]
hi @wheeleryanli, hope you found a way to resolve the issue. If not you can try this:

- **Check Your Eligibility:** Ensure that you still meet the eligibility requirements for GitHub Education benefits. Sometimes, benefits may need to be reapplied for if your status (e.g., student enrollment) changes.
- **Review Your Application:** Look at the application you submitted. Make sure all information is accurate and complete. Errors or missing information can lead to issues with your application.
- **Contact Support:** If everything seems correct and you're still facing issues, reach out to GitHub Support. They can provide specific insights into why you need to reapply and help resolve any problems.
- **Stay Updated:** Keep an eye on your GitHub account and email for any updates regarding your application status.
Cheers!

**Comment by samus-aran** [comment]
Hi @wheeleryanli 👋 

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.
We apologize for any inconvenience this may cause.

Thank you for your understanding.
